lilac2/building.py

```diff
@@ -25,7 +25,7 @@ def __init__(self, msg: str) -> None:
     self.msg = msg
 
 def lilac_build(
-  mod: LilacMod, build_prefix: Optional[str] = None,
+  mod: LilacMod, repo, build_prefix: Optional[str] = None,
```

Comment:
Type annotation.

Reply:
Using `repo: Repo` here requires importing lilac2/repo.py from lilac2/building.py. It does not work as lilac2/building.py already imports lilac2/repo.py.

Reply:
See d8388dea662348745bbd1f78d6d923993fff48ac.

Reply:
Aha, brilliant!